composer.json

```diff
@@ -13,15 +13,15 @@
   "require": {
     "php": ">=7.2",
     "ext-json": "*",
-    "roave/security-advisories": "dev-master",
     "silinternational/psr3-adapters": "^1.1 || ^2.0",
-    "simplesamlphp/simplesamlphp": "~1.17.7"
+    "simplesamlphp/simplesamlphp": "~1.17.7 || ^1.18"
```

Comment:
I think it's the same as `^1.17.7` (which is `>=1.17.7 <2.0.0`)

`~1.17.7` = `>=1.17.7 <1.18.0`
and
`^1.18` = `>=1.18.0 <2.0.0`
is
`>=1.17.7 <2.0.0`

Reply:
Hmm, sorry about that @Schparky, my mistake. @Great-Antique's comment is correct: I suggested a less-than-ideal version constraint. I’m so used to using the carat (^) that I forgot that we often use the tilde for our SSP-related code (since SimpleSAMLphp can introduce changes that are backwards breaking for the module developer even in between major releases).

I would suggest using `~1.17.7 || ~1.18.0` or `^1.17.7` instead.

Reply:
Not your fault. I shouldn't have blindly copied your suggestion.